README.md

```diff
@@ -1,2 +1,213 @@
+
+    Next, we need to obtain an access token and a refresh token to invoke the Microsoft Graph API.
+    First, in a new browser enter this URL by replacing the \<MS_CLIENT_ID> with the application ID.
+
+    ```
+    https://login.microsoftonline.com/common/oauth2/v2.0/authorize?response_type=code&client_id=<MS_CLIENT_ID>&redirect_uri=https://login.microsoftonline.com/common/oauth2/nativeclient&scope=Files.ReadWrite openid User.Read Mail.Send Mail.ReadWrite offline_access
+    ```
+
+    This may prompt you to enter the username and password for signing into  Azure Portal App.
+
+    Once the username password pair successfully entered this will give a URL like follows on the browser address bar.
+
+    https://login.microsoftonline.com/common/oauth2/nativeclient?code=M95780001-0fb3-d138-6aa2-0be59d402f32
+
+    Copy the code parameter (M95780001-0fb3-d138-6aa2-0be59d402f32 in the above example) and in a new terminal enter the following CURL command with replacing the \<MS_CODE> with the code received from the above step. The \<MS_CLIENT_ID> and \<MS_CLIENT_SECRET> parameters are the same as above.
+
+    ```
+    curl -X POST --header "Content-Type: application/x-www-form-urlencoded" --header "Host:login.microsoftonline.com" -d "client_id=<MS_CLIENT_ID>&client_secret=<MS_CLIENT_SECRET>&grant_type=authorization_code&redirect_uri=https://login.microsoftonline.com/common/oauth2/nativeclient&code=<MS_CODE>&scope=Files.ReadWrite openid User.Read Mail.Send Mail.ReadWrite offline_access" https://login.microsoftonline.com/common/oauth2/v2.0/token
+    ```
+
+    The above CURL command should result in a response as follows,
+    ```
+    {
+    "token_type": "Bearer",
+    "scope": "Files.ReadWrite openid User.Read Mail.Send Mail.ReadWrite",
+    "expires_in": 3600,
+    "ext_expires_in": 3600,
+    "access_token": "<MS_ACCESS_TOKEN>",
+    "refresh_token": "<MS_REFRESH_TOKEN>",
+    "id_token": "<ID_TOKEN>"
+    }
+    ```
+
+    Set the path to your Ballerina distribution's trust store in the <TURST_STORE_PATH>. This is by default located in the following path.
+
+    $BALLERINA_HOME/distributions/jballerina-<BALLERINA_VERSION>/bre/security/ballerinaTruststore.p12
+
+    The default TRUST_STORE_PASSWORD is set to "ballerina".
+
+    WORK_BOOK_NAME, WORK_SHEET_NAME, and TABLE_NAME corresponds to workbook file name (without the .xlsx extension), worksheet name, and the table name respectively. Makesure you create a workbook with the same WORK_BOOK_NAME on Microsoft OneDrive before using the connector.
+
+3. Create a new Ballerina project by executing the following command.
+
+	```shell
+	<PROJECT_ROOT_DIRECTORY>$ ballerina init
+	```
+
+4. Import the Microsoft Graph connector to your Ballerina program as follows. The following sample program creates a new worksheet on an existing workbook on Microsoft OneDrive. Prior running this application please create a workbook on your Microsoft OneDrive account
```

Comment:
```suggestion
4. Import the Microsoft Graph connector to your Ballerina program as follows. 

The following sample program creates a new worksheet on an existing workbook on Microsoft OneDrive. Prior to running this application, create a workbook on your Microsoft OneDrive account
```
Do not use "please" in technical documentation.

Reply:
Ok, Done